src/CredentialsLoader.php

```diff
@@ -28,7 +28,7 @@
  */
 abstract class CredentialsLoader implements FetchAuthTokenInterface
 {
-  const TOKEN_CREDENTIAL_URI = 'https://www.googleapis.com/oauth2/v3/token';
+  const TOKEN_CREDENTIAL_URI = 'https://www.googleapis.com/oauth2/v4/token';
```

Comment:
What implications does this have? I don't feel comfortable blindly updating this API version.